### PR TITLE
Fix toggle size defaults

### PR DESCRIPTION
**Pull request by evgenoid**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Screenshot 2023-12-15 at 14 28 48

## What is the new behavior?
Screenshot 2023-12-15 at 14 30 17

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/form/Toggle/Toggle.module.css [hunk 1]
```diff
@@ -27,35 +27,35 @@
   }
 
   &.small {
-    height: calc(var(--toggle-width, 35px) / 2);
+    height: calc(var(--toggle-height, 35px) / 2);
```
Comment (amcdnl) [inline src/form/Toggle/Toggle.module.css:30]: Don't we want the defaults here?

Comment (evgenoid) [reply src/form/Toggle/Toggle.module.css:30]: We want. Was a mistake in variable uses

File: src/form/Toggle/Toggle.module.css [hunk 2]
```diff
@@ -27,35 +27,35 @@
...
   &.medium {
-    height: calc(var(--toggle-width, 35px) / 1.5);
+    height: calc(var(--toggle-height, 35px) / 1.5);
     width: calc(var(--toggle-width, 55px) / 1.5);
     padding: calc(var(--toggle-spacing) / 1.5);
 
     .handle {
-      height: calc(var(--toggle-width, 25px) / 1.5);
-      width: calc(var(--toggle-width, 25px) / 1.5);
+      height: calc(var(--toggle-handle-size) / 1.5);
```
Comment (amcdnl) [inline src/form/Toggle/Toggle.module.css:46]: What about defaults here?

Comment (evgenoid) [reply src/form/Toggle/Toggle.module.css]: Done

**Review by amcdnl — APPROVED** [review]

**Pull request merged**